browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -53,6 +53,8 @@ void AIChatUIPageHandler::SubmitHumanConversationEntry(
   // TODO(nullhook): Abstract prompt injection cleanups to a central place
   base::ReplaceSubstringsAfterOffset(&input_copy, 0, ai_chat::kHumanPrompt, "");
   base::ReplaceSubstringsAfterOffset(&input_copy, 0, ai_chat::kAIPrompt, "");
+  base::ReplaceSubstringsAfterOffset(&input_copy, 0, "<article>", "");
+  base::ReplaceSubstringsAfterOffset(&input_copy, 0, "</article>", "");
```

Comment:
Should we strip history and question tags from the input rather than article?

Reply:
True. Should we sanitise out all _privileged_ tags for both article and subsequent outputs?